### PR TITLE
Update Software

### PR DESCRIPTION
**Pull request by zmb3**
Update to deploy the latest versions.  Including:

- Ops Manager 2.3
- PAS: 2.3
- Healthwatch 1.3.2

There were a few larger changes necessary than typical for this update, as Ops Manager 2.3 expects to be configured via API only.  In order to accomplish this, we pulled in the latest version of github.com/pivotal-cf/om.  The govendor tool was unable to resolve the required dependencies in order to make this upgrade, so the project was converted to a Go module (which requires Go 1.11+).  As a bonus, the entire vendor directory goes away but we still have reproducible builds.

### CODE REVIEW

**Comment by mattysweeps** [comment]
@zmb3 I just saw this today (I don't know how I missed it until now)

I am testing this with our internal CI server now.
I'll post back if I have any questions.

File: ci/tasks/utils.sh [hunk 1]
```diff
@@ -54,6 +54,17 @@ set_gcloud_config() {
 	gcloud auth activate-service-account --key-file=${key}
 }
 
+build_go() {
+  my_dir=$(dirname "$(readlink -f "$0")")
+  release_dir="$( cd ${my_dir} && cd ../.. && pwd )"
+  omg_dir="${release_dir}/src/omg-cli"
+
+  pushd omg_dir
```
Comment (mattysweeps) [inline ci/tasks/utils.sh]:
```suggestion
  pushd ${omg_dir}
```

**Comment by mattysweeps** [comment]
I've created the branch `up-sw-patches` to extend on this branch.

Currently I'm running into an error installing the stackdriver nozzle.
I believe this is because it is trying to install before the elastic runtime.

I wonder if that tile updated it's name to something other than `cf`, as the nozzle has `cf` as a product dependency.

**Comment by mattysweeps** [comment]
@zmb3 just an update:

Both @rkoster and myself have run into an issue with credhub. It seems credhub isn't connecting to the internal mysql database: https://gist.github.com/mattysweeps/1cb785245bf14d5b467feaef8186f604

Just wondering, were you able to successfully install PCF on your branch? or are you still in the process of testing?

**Comment by zmb3** [comment]
@mattysweeps I did succesfully deploy with my branch.  Not sure exactly what’s going on here.

**Comment by mattysweeps** [comment]
@zmb3 I just tried installing from this branch and I ran into the same problems.

https://gist.github.com/mattysweeps/8a921134c49be0c0687773aecc76b520
^ This failure is due to an issue with the Stackdriver nozzle. @rkoster stated this is due to the nozzle not adding in its metadata the correct dependency on the cf tile. https://github.com/cloudfoundry-community/stackdriver-tools/issues/216

He also authored a temporary fix for that issue here; https://github.com/cf-platform-eng/gcp-pcf-quickstart/commit/9f139e4b12a9b99067cc4dc653c1bf708e1dbb82

Maybe there is something different between our deployments?

**Comment by mattysweeps** [comment]
@zmb3 out of curiosity did you deploy a small runtime or a full runtime? I've been deploying small runtimes.

**Comment by zmb3** [comment]
Small footprint.

I might be able to spend a few minutes looking into this late next week, but if it's too much trouble feel free to close this.  I ended up writing some alternative tooling that meets my needs a little better.

**Comment by mattysweeps** [comment]
Closing this since we merged #51, which included commits from the `update-sw` branch

**Pull request closed without merging**